example/flask/README.md

```diff
@@ -0,0 +1,55 @@
+# Bugsnag Flask demo
+
+This Flask application demonstrates how to use Bugsnag with the Flask web
+framework for Python. Before testing it, open up the `server.py`
+file and configure your API key.
+
+```
```

Comment:
You can use the following to add syntax highlighting to the rendered code-block:

    ```python